notation.go

```diff
@@ -347,12 +347,12 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, ve
 		return ocispec.Descriptor{}, nil, ErrorSignatureRetrievalFailed{Msg: fmt.Sprintf("user input digest %s does not match the resolved digest %s", ref.Reference, artifactDescriptor.Digest.String())}
 	}
 
+	var verificationSucceeded bool
 	var verificationOutcomes []*VerificationOutcome
+	var verificationFailedErr error = ErrorVerificationFailed{}
```

Comment:
It seems `verificationFailedErr` is initialized and returned but never assigned.

Reply:
Yes, this is also the behavior in v1.0.0. This is the general error on verification failure. When printing its error message, the output is `signature verification failed`.